### DISCUSSION TITLE
Playlist and/or collections from folders in library

### DISCUSSION THREAD

**Discussion by dhowe** [meisnate12/Plex-Meta-Manager#1640] [Q&A]
Wondering if PMM supports creating playlists and/or collections from folders ? Thanks!

**Comment by chazlarson** [comment]
PMM only works with Plex libraries, not local folders.

If this is not what you mean, you'll need to add more details.

**Comment by dhowe** [comment]
My mistake -- I was thinking from the perspective of library folders living on a server. But you are correct, I mean from folders in a Plex library (I've updated the title)

**Comment by chazlarson** [comment]
Sure, you can filter on file path.

<https://metamanager.wiki/en/nightly/metadata/filters.html#string-filters>

```
collections:
  Movies in that one folder:
    plex_all: true
    filters:
      filepath: Folder Name
```

**Comment by dhowe** [comment]
thanks, will give a try
